packages/@lwc/features/src/index.ts

```diff
@@ -19,6 +19,7 @@ const features: FeatureFlagMap = {
     ENABLE_FROZEN_TEMPLATE: null,
     ENABLE_LEGACY_SCOPE_TOKENS: null,
     ENABLE_FORCE_SHADOW_MIGRATE_MODE: null,
+    ENABLE_EXPERIMENTAL_SIGNALS: null,
```

Comment:
@nolanlawson this is the flag I'm planning to use to prevent anyone from using signals, do you know if there's anything else I should add to gate the feature?

Reply:
Let me help you offline. :)